### PR TITLE
Medical Treatment - Add new Grave model

### PR DESCRIPTION
**Pull request by Marstruc**
**When merged this pull request will:**
- Add a new grave model
- Changes instances of `Land_Grave_Dirt_F` to `Land_ACE_Grave`

### IMPORTANT

- [x] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Review by jonpas — COMMENTED** [review]
We prefix our models, should start with `ACE_`. `ACE_Grave` would be fine, `ACE_Grave_[dirt/...]` also.

**Comment by jonpas** [comment]
Sorry, what I meant was the class name.

**Comment by LinkIsGrim** [comment]
Screenshots?

File: addons/medical_treatment/functions/fnc_placeInGrave.sqf [hunk 1]
```diff
@@ -23,9 +23,9 @@ if ((alive _patient) && {GVAR(allowGraveDigging) < 2}) exitWith {
     [LSTRING(bodybagWhileStillAlive)] call EFUNC(common,displayTextStructured);
 };
 
-private _graveClassname = missionNameSpace getVariable [QGVAR(graveClassname), "Land_Grave_dirt_F"];
-// Land_Grave_dirt_F needs to be rotated 90 degrees to line up with the body
-private _graveRotation = missionNameSpace getVariable [QGVAR(graveRotation), 90]; 
+private _graveClassname = missionNameSpace getVariable [QGVAR(graveClassname), "Land_ACE_Grave"];
+// Land_Grave_dirt_F needed to be rotated 90 degrees to line up with the body, Land_ACE_Grave does not.
```
Comment (johnb432) [inline addons/medical_treatment/functions/fnc_placeInGrave.sqf:27]:
```suggestion
// Land_Grave_dirt_F needs to be rotated 90 degrees to line up with the body, Land_ACE_Grave does not.
```

**Comment by Drofseh** [comment]
Just `ACE_Grave` is fine for the classname I think. Don't need the `Land_` prefix

File: addons/medical_treatment/functions/fnc_placeInGrave.sqf (hunk 1 above)
Comment (Drofseh) [reply addons/medical_treatment/functions/fnc_placeInGrave.sqf:27]: This new model might not need the rotation, in which case the entire comment can be removed instead.

File: addons/medical_treatment/CfgVehicles.hpp [hunk 2]
```diff
@@ -33,6 +33,13 @@ class CfgVehicles {
         };
     };
 
+    // Grave vehicle
+	class Land_Grave_dirt_F;
```
Comment (PabstMirror) [inline addons/medical_treatment/CfgVehicles.hpp:37]:
```suggestion
    class Land_Grave_dirt_F;
```

File: addons/medical_treatment/functions/fnc_placeInGrave.sqf (hunk 1 above)
Comment (Marstruc) [reply addons/medical_treatment/functions/fnc_placeInGrave.sqf:27]: I'm not a programmer. I took a cursory look and it looked like `_graveRotation` was still needed as a parameter. I can remove the comment though, yeah.

**Comment by Marstruc** [comment]
> Screenshots?

Here ya go

File: addons/medical_treatment/data/model.cfg [hunk 3]
```diff
@@ -58,6 +58,8 @@ class CfgModels {
     class IVBag_500ml: IVBagBase {};
     class IVBag_1000ml: IVBagBase {};
 
+    class ACE_grave: IVBagBase {};
```
Comment (LinkIsGrim) [inline addons/medical_treatment/data/model.cfg:61]: I'm not familiar with models.

@PabstMirror does this look okay?

**Review by LinkIsGrim — APPROVED** [review]
LGTM other than the model.cfg comment.

File: addons/medical_treatment/data/model.cfg (hunk 3 above)
Comment (PabstMirror) [reply addons/medical_treatment/data/model.cfg:61]: I think so,
I know it needs `sections[] = {"camo"};`
but it's getting that through inheritance

Comment (Marstruc) [reply addons/medical_treatment/data/model.cfg:61]: Afaik, it's completely safe to do that.

**Pull request merged**